DESCRIPTION

```diff
@@ -39,7 +39,7 @@ License: MIT + file LICENSE
 Depends: R (>= 3.1.2)
 Imports:
     assertthat (>= 0.2),
-    dplyr (>= 0.7.0),
+    dplyr (>= 0.8.0),
```

Comment:
dplyr 0.8.1 _also_ had many breaking changes; maybe set to that.

Reply:
Set to 0.8.2